<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorSelect</class>
 <widget class="QDialog" name="SensorSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕코딩</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox::indicator {
     width: 30px;
     height: 30px;
}

QCheckBox::indicator:checked
{
	image: url(:images/checkbox_checked.png);
}
QCheckBox::indicator:unchecked
{
	image: url(:images/checkbox_normal.png);
}

QRadioButton::indicator {
     width: 30px;
     height: 30px;
}

QRadioButton::indicator:checked
{
	image: url(:images/radiobutton_checked.png);
}
QRadioButton::indicator:unchecked
{
	image: url(:images/radiobutton_normal.png);
}
</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>510</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>380</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>1021</width>
     <height>1031</height>
    </rect>
   </property>
   <property name="title">
    <string>측정기기</string>
   </property>
   <widget class="QPushButton" name="TN">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TN</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="TOC">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TOC</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="TP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>TP</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PH">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>PH</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SS">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>SS</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="COD">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>COD</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>630</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>채수기 모델</string>
    </property>
    <widget class="QRadioButton" name="Io">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Io</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">SamModelGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="Hanchang">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hanchang</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">SamModelGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="Watersam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Watersam</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">SamModelGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="Year100">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Year100</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">SamModelGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="Korbi">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Korbi</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">SamModelGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="Ilho">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ilho</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">SamModelGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="Dongil">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>120</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dongil</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">SamModelGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QCheckBox" name="Analog_tn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>컨버터 사용</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Analog_tp">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>컨버터 사용</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Analog_cod">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>140</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>컨버터 사용</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Analog_ph">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>컨버터 사용</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Analog_ss">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>컨버터 사용</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Analog_toc">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TOC10</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>630</y>
      <width>361</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>유량계</string>
    </property>
    <widget class="QCheckBox" name="Analog_flow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>181</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>컨버터 사용</string>
     </property>
    </widget>
    <widget class="NumEdit" name="num">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>61</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="rate">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>141</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>순시 유량</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="flowNew">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>9</y>
       <width>121</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NewFlow</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">FlowGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>100</y>
       <width>61</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>갯수</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="flowOld">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>9</y>
       <width>91</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Flow</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">FlowGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>81</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>적산배율</string>
     </property>
    </widget>
    <widget class="NumEdit" name="ratio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>111</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="WaterTemp_ss">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>수온</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Gas_toc">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>230</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이송가스</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">numedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>SensorSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>800</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="FlowGroup"/>
  <buttongroup name="SamModelGroup"/>
 </buttongroups>
</ui>
